udata_front/theme/gouvfr/datagouv-components/src/components/button.less

```diff
@@ -2,4 +2,17 @@
 
 .datagouv-components {
     @import "../../../assets/less/components/buttons.less";
+
+    .fr-btn.fr-btn--xs[class*=" fr-icon-"]:not([class*=fr-btn--icon-]) {
```

Comment:
I think we can move this `.fr-btn--xs` class to buttons.less, if we refactor the color and hover outside of it. Or it can be renamed and added to QualityComponentInline as scoped less code